ShenzhenMod/App.config

```diff
@@ -1,27 +1,39 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <configSections>
-    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net"/>
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" />
+    
+    <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
```

Comment:
Looks like Visual Studio has added some spurious EntityFramework references here (and in the section below). Might be best to remove them (or revert this file if there are no other changes!).

Reply:
Whoops! I had added EntityFramework when I was attempting to get WebRequest working in a using block, with no success. It looks like those references stuck around even after removing the package.